docs/config/lua/config/window_background_gradient.md

```diff
@@ -68,6 +69,26 @@ Gradients are implemented using the `colorgrad` crate.
 Take a look at <https://github.com/mazznoer/colorgrad-rs#using-web-color-format>
 for some usage examples and additional information about gradients.
 
+## Linear gradient:
+
+*Since: nightly builds only*
+
+A linear gradient follows a linear path across the window. It can be rotated
+around the window center. The start angle is at the center left and rotated
```

Comment:
The "start angle is at the center left" portion of this feels hard to understand.

Can we say something like this, adjusted to match up with the actual implementation, so that it is easier to see what a given value might result in:

> `0` degrees is equivalent to `Horizontal` with the gradient moving from left to right. `90` degrees is equivalent to `Vertical` with the gradient moving from top to bottom. `180` degrees is equivalent to `Horizontal` but with the gradient moving from right to left.  `270` degrees is equivalent to `Vertical` but with the gradient going from bottom to top.
> Negative degrees are equivalent to going counter clockwise, so `-45` is equivalent to `270` degrees and results in a gradient that moving from the top left corner down to the bottom right corner.